### ISSUE TITLE
Missing files from the latest release

### ISSUE THREAD

**Issue by rockingdice** [robertying/openssl-curl-android#4]
I cannot find files for curl armeabi-v7a in the latest release:
openssl 1.1.1h & curl 7.72.0.
Please check the build zip file.

**Comment by rockingdice** [comment]
I found running the script will get an error:
`
../libtool: line 1720: /Users/li9s/Library/Android/sdk/ndk/21.1.6352462//toolchains/llvm/prebuilt/darwin-x86_64/bin/armv7a-linux-androideabi-ar: No such file or directory
make[2]: *** [libcurl.la] Error 127
make[1]: *** [all] Error 2
make: *** [all-recursive] Error 1
`
FYI:
https://github.com/android/ndk/issues/1324

**Comment by robertying** [comment]
This was caused by the naming of 32 bit arm toolchain.

It says in the doc:

```
Note: For 32-bit ARM, the compiler is prefixed with armv7a-linux-androideabi, but the binutils tools are prefixed with arm-linux-androideabi. For other architectures, the prefixes are the same for all tools.
```

**Comment by robertying** [comment]
Fixed by 9ab10f09f5e613025cac95df5f02ea6901971536
